coreos-ostree-importer/send-ostree-import-request.py

```diff
@@ -43,6 +44,10 @@ def parse_args():
         required=True,
         help="bucket and prefix to S3 builds/ dir",
     )
+    parser.add_argument(
+        "--arch",
+        help="target architecture",
+    )
```

Comment:
Minor/optional: seems like this would fit fine on one line. Other ones are split to satisfy pep8.